Networking/NetworkingTests/Mapper/CustomerMapperTests.swift

```diff
@@ -0,0 +1,82 @@
+import XCTest
+@testable import Networking
+
+/// CustomerMapper Unit Tests
+///
+class CustomerMapperTests: XCTestCase {
+
+    /// Dummy Site ID.
+    ///
+    private let dummySiteID: Int64 = 123
+
+    /// Local file that holds Customer data representing the API endpoint
+    ///
+    private let filename: String = "customer"
+
+    /// Verifies that the Customer object can be mapped fron the Encoded data
+    ///
+    func test_Customer_is_mapped_from_encoded_data() {
+        // Given
+        let mapper = CustomerMapper(siteID: dummySiteID)
+        guard let data = Loader.contentsOf(filename) else {
+            XCTFail("customer.json not found")
+            return
+        }
+
+        // When
+        let customer = try? mapper.map(response: data)
+
+        // Then
+        XCTAssertNotNil(mapper)
+        XCTAssertNotNil(customer)
+        XCTAssertNotNil(customer?.customerID)
+        XCTAssertNotNil(customer?.email)
+        XCTAssertNotNil(customer?.firstName)
+        XCTAssertNotNil(customer?.lastName)
+        XCTAssertNotNil(customer?.billing)
+        XCTAssertNotNil(customer?.shipping)
```

Comment:
You can remove these asserts at least, or even this whole test, as they are covered by the `correctly_parsed` test

Reply:
Gotcha! Done at 2fe9688ec26da1d8c0186bdd67290d3b15ccb38d. I left the rest of the test so we can test the difference between if the data can be loaded from the local file vs if the values are correctly parsed. I see the `correctly_parsed` test also includes that the data must be loaded first in order to work, but it felt right to keep them separate. Happy to remove it entirely in any case 👍